Yoast/wordpress-seo
# Improve translation commments in yoastseo package

## Context


* The change is made in line with [the WP translation guide](https://developer.wordpress.org/apis/internationalization/internationalization-guidelines/#descriptions) where the word translations in comments for translations should be non-capitalized. The previously incorrect syntax resulted in the comment not being displayed on translationpress.com which is problematic for translators who should be able to see it.

## Summary


This PR can be summarized in the following changelog entry:

* Changes `Translators` to non-capitalized `translators` in files within the _yoastseo_ package.

## Relevant technical choices:

* 

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

To check if the solution work for non-Free assessments/code:
* Build the corresponding add-on by running grunt build
* Check the .pot file
* Confirm that now the translator comments start with `translators` instead of `Translators`

The translator comments for non-Free assessments are expected to appear in Translationpress. Though maybe we can only check this once the PR is merged to trunk and Premium has been bumped to the Free version that contains this fix.


#### Relevant test scenarios


### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

* Open translations for Free assessments in Translationpress and confirm that the translator comments appear.

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

*

## UI changes


## Other environments


## Documentation


## Quality assurance


## Innovation


Fixes #https://github.com/Yoast/lingo-other-tasks/issues/184
